dynamodb-enhanced/runtime/src/main/java/io/quarkus/amazon/dynamodb/enhanced/runtime/DynamoDbEnhancedBuildTimeConfig.java

```diff
@@ -0,0 +1,29 @@
+package io.quarkus.amazon.dynamodb.enhanced.runtime;
+
+import java.util.List;
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigDocSection;
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "dynamodbenhanced", phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
+public class DynamoDbEnhancedBuildTimeConfig {
+
+    /**
+     * Extension of the DynamodbEnhancedClient
+     */
+    @ConfigItem
+    @ConfigDocSection
+    Optional<List<DynamodbEnhancedClientExtension>> clientExtensions = Optional
+            .of(List.of(DynamodbEnhancedClientExtension.VersionedRecordExtension,
+                    DynamodbEnhancedClientExtension.AutoGeneratedTimestampRecordExtension));
```

Comment:
I'm not sure we should enable the extensions by default. Also that wouldn't be the way to do it.

Reply:
You are right! I removed this.